gradle/libs.versions.toml

```
@@ -0,0 +1,110 @@
+# keep sorted please, use Edit -> Sort Lines
+
+[versions]
+accompanist = "0.28.0"
+agp = "8.1.1"
+androidCompileSdk = "34"
+androidMinSdk = "21"
+androidTargetSdk = "32"
+androidx-lifecycle = "2.6.1"
+androidx-navigation-compose = "2.5.1"
+appyx = "2.0.0-alpha10"
```

```suggestion
appyx = "2.0.0-alpha09"
```

The latest version is 09, right? I can't download the 10

---

Good catch! Reverted!